ci-operator/jobs/openshift/assisted-service/openshift-assisted-service-master-presubmits.yaml

```diff
@@ -14,6 +14,7 @@ presubmits:
       pj-rehearse.openshift.io/can-be-rehearsed: "true"
     name: pull-ci-openshift-assisted-service-master-e2e-metal-assisted
     rerun_command: /test e2e-metal-assisted
+    run_if_changed: ^(cmd/.*|deploy/.*|hack/.*|internal/.*|pkg/.*|tools/.*|Makefile|go\.mod|go\.sum)$
```

Comment:
maybe it should be a black list instead?

Reply:
Harder to follow a denylist. what's wrong with this list?

Reply:
prow doesn't support denylist
let's try this out, although I will feel better with ``**/*.go`` if it's possible adding to the list

Reply:
But then you will run for subsystem as well

Reply:
yes, you're right